### PR TITLE
[KAS-10]write tests sign up view controller

### PR DESCRIPTION
**Pull request by Harrisali93**
### Jira Story Link:

[KAS-10](https://leezeit.atlassian.net/browse/KAS-10?atlOrigin=eyJpIjoiZjAzYWY4ZmJjYmNmNDM4NTkzMDUwY2Q1Y2QwODAzNDAiLCJwIjoiaiJ9)

**Description**:

1) Unit test for  textfield method user cannot add number in name Textfield
2) Unit test  for textfield method user is adding proper email in emailTextfield

### CODE REVIEW

File: LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     
+     func testTextFieldDoesntAcceptSymbols() {
+         XCTAssertFalse(viewController.checkTextField(string: "#"))
+     }
+     
+    // func testPassWordIsSecureEntry() throws {
+       //  let passwordTest =  try? //XCTUnwrap(viewController.passwordTextField)
+        // XCTAssert((passwordTest!.isSecureTextEntry) )
+   //  }
+}
```
Comment (RiffatiKhan) [inline LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift]: Why this function is in comment.

File: LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+     }
+     
+     func testTextFieldDoesntAcceptSymbols() {
+         XCTAssertFalse(viewController.checkTextField(string: "#"))
+     }
+     
+    // func testPassWordIsSecureEntry() throws {
+       //  let passwordTest =  try? //XCTUnwrap(viewController.passwordTextField)
+        // XCTAssert((passwordTest!.isSecureTextEntry) )
+   //  }
```
Comment (RiffatiKhan) [inline LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift]: I have uncomment  this and test is not working properly.
it says test failed.

**Review by RiffatiKhan — COMMENTED** [review]
Resolved these comments.

File: LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift (hunk 1 above)
Comment (Harrisali93) [reply LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift]: resolved

File: LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift (hunk 2 above)
Comment (Harrisali93) [reply LeezeIt/LeezeItTests/CreateAccountViewControllerTests.swift]: resolved

**Review by RiffatiKhan — DISMISSED** [review]
Looks good

**Review by arfhan786 — DISMISSED** [review]

**Review by RiffatiKhan — APPROVED** [review]
Approved

**Review by arfhan786 — APPROVED** [review]

**Pull request merged**